src/DefaultBuilder/src/WebApplicationAuthenticationBuilder.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Authentication;
+
+internal class WebApplicationAuthenticationBuilder : AuthenticationBuilder
+{
+    public bool IsAuthenticationConfigured { get; private set; }
```

Comment:
I think its worth moving the concept of whether any schemes were configured up to the base AuthenticationBuilder.

Reply:
Tracked in https://github.com/dotnet/aspnetcore/issues/41888.